Vagrantfile

```diff
@@ -121,6 +121,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   config.vm.network "forwarded_port", guest: 9991, host: host_port, host_ip: host_ip_addr
+  config.vm.network "forwarded_port", guest: 9994, host: 9994, host_ip: host_ip_addr
```

Comment:
We shouldn't need to do this; if we configure things right, webpack requests should be going to port 9991 and then be being proxied to port 9994 by `tools/run-dev.py`.

Reply:
As far as I can tell, this is the best possible solution to make HMR work with the Vagrant VM. I tried to forward the calls to sockjs-node using the config file but for some reason my attempts were unsuccessful. Also, take a look at https://bugzilla.mozilla.org/show_bug.cgi?format=default&id=1362443 someone else from mozilla came to the same conclusion i think.